### PR TITLE
new "Book" theme

### PR DESCRIPTION
**Pull request by s3bk**
The idea is to add a theme that mimics traditional typesetting for books.

Preview:
[Rust Reference](https://rynx.org/sebk/rust-reference)
[Rust Book](https://rynx.org/sebk/rust-book)

### CODE REVIEW

File: src/theme/css/book.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    position: absolute;
+    right: 0;
+}
+    
+#menu-bar-sticky-container {
+    background-color: rgba(255, 255, 255, 0.8) !important;
+}
+
+@media print {
+    .bottons {
```
Comment (Havvy) [inline src/theme/css/book.css]: `.buttons`?

Comment (s3bk) [reply src/theme/css/book.css]: oops.

**Comment by Dylan-DPC** [comment]
Thanks

**Comment by ehuss** [comment]
This PR has broken formatting for all of the themes.  All fonts have changed to `Source Serif Pro`.  This also makes the font story more difficult by adding more fonts.

I can't really tell what the intention was, but I don't think the included `book.css` is a good idea.  It should probably use variables like all other themes.

**Comment by Dylan-DPC** [comment]
Reverting this

**Comment by Dylan-DPC** [comment]
Update: can't revert it because GH considers it might be changed.

**Comment by s3bk** [comment]
Oops. I will try to look into that today.

**Pull request merged**